File: p20/p20.erl
#!/usr/bin/env escript
%% -*- erlang -*-
%%! -smp enable -sname p20
% vim:syn=erlang

-mode(compile).

main(_) ->
    io:format("Answer: ~p ~n", [lists:foldl(fun(E,A)-> A + E - 48 end, 
                                            0, integer_to_list(fac(100)))]).

fac(1) -> 1;
fac(N) -> N*fac(N-1).

File: p20/p20.py
#!/usr/bin/python

import math

def digits(n):
    s = 0
    while n > 0:
        s = s + (n % 10)
        n = n / 10
    return s

print "Answer %s" % digits(math.factorial(100))

print "Answer (onliner) %s " % sum(map(int, str(math.factorial(100))))
